src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -516,8 +516,8 @@ def get_pv(pv_name=None, pv_uuid=None, pv_tags=None, pvs=None):
 #
 #############################
 
-#TODO add vg_extent_size here to have that available in VolumeGroup class
-VG_FIELDS = 'vg_name,pv_count,lv_count,snap_count,vg_attr,vg_size,vg_free,vg_free_count'
+VG_FIELDS = 'vg_name,pv_count,lv_count,vg_attr,vg_extent_count,vg_free_count,vg_extent_size'
+VG_CMD_OPTIONS = ['--noheadings', '--readonly', '--units=b', '--nosuffix', '--separator=";"']
```

Comment:
Maybe name `VG_CMD_OPTIONS` as something more generic, since these options would also normally apply to `pvs` and `lvs` commmands?

Reply:
:+1: Let's do that if/when we use those cmd options for other lvm commands.